lib/zendesk_api/middleware/response/sanitize_response.rb

```diff
@@ -0,0 +1,21 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitizeResponse < Faraday::Response::Middleware
+        def on_complete(env)
+          to_utf8!(env[:body])
+        end
+
+        def to_utf8!(str)
+          str.force_encoding(Encoding::UTF_8)
+          if str.respond_to?(:scrub)
```

Comment:
to keep these consistent how about `return str if str.valid_encoding?` before the `if`